about-me.md

```diff
@@ -1,9 +1,9 @@
 # About Me
 
-![Picture of me]()
+![Picture of me](https://avatars2.githubusercontent.com/u/49763671?s=460&v=4)
 
-Hi, I'm Charlene Bernard. I'm a ________ at *__________*.
+Hi, I'm Alex Hoover. I'm a UX Designer at *USDS*.
```

Comment:
Are you sure you want to change your name? I kind of like Charlene for you.

Reply:
It is a lovely name.